### ISSUE TITLE
LaTeX display problem in published page

### ISSUE THREAD

**Issue by aweimeow** [hackmdio/hackmd#252]
hi all,

I met this problem in published page, 
I wrote
```
> $2 * (blockSize)^2 * wordSize = L1cachesize$
```

and preview seems good, but in published page it becomes:
```
\(2 * (blockSize)^2 * wordSize = L1cachesize\)
```

Picture for this issue:
* Preview looks good
![_004](https://cloud.githubusercontent.com/assets/5120010/20342658/4c91edec-ac26-11e6-9564-5d5d9870f742.png)

* but published page broke
![_005](https://cloud.githubusercontent.com/assets/5120010/20342684/6190c830-ac26-11e6-90f7-25f1c26ca8ee.png)

**Comment by jackycute** [comment]
Bug confirmed, inspect later.

**Comment by jackycute** [comment]
OK, so that was a race condition between blockquote and mathjax processing.
Fixed in 80ee507951914aef9df62d05d7d482cbb44c4729

Thanks again.

**Comment by aweimeow** [comment]
Thanks :smiley:
